components/eamxx/src/physics/share/physics_constants.hpp

```diff
@@ -107,6 +107,10 @@ struct Constants
   static constexpr int VTABLE_DIM1    = 10;
   static constexpr int MU_R_TABLE_DIM = 150;
 
+  // Turbulent Mountain Stress constants
+  static constexpr Scalar orocnst = 1;     // Converts from standard deviation to height [ no unit ]
+  static constexpr Scalar z0fac   = 0.075; // Factor determining z_0 from orographic standard deviation [ no unit ]
```

Comment:
I found these namelist defaults in eam and made them physics constants. Do we need these as variables? That's an easy switch, perhaps they can be AP params.